nginx/templates/nginx.conf

```diff
@@ -17,17 +16,27 @@
 
 
 user              nginx;
-worker_processes  1;
+# the following has been parametrized via map.jinja in the
+# for loop (see above)
+# worker_processes  1;
```

Comment:
I'm guessing this means that worker_processes *must* be set in map.jinja? In which case, see the above comment. If so, another way would be to 'if' get the setting and default to worker_processes  1 if its not found.